### PR TITLE
Login registration

### PR DESCRIPTION
**Pull request by spaceytato**
### Team Monkey pull request checklist (Jungle edition)

-   [x] Remember to format your code! (`npm run prettier && npm run eslint`)
-   [x] Remember to run the tests! (`npm run test`)
-   [x] Don't forget comments and docstrings!
-   [x] Is it a breaking change? Remember to mention that!

### Description

Login and Registration styling. The "Forgot my Password" and "Remember Me" don't currently work. Some tweaking to styling may occur further in the process. 
🐵🐒🙊🙉🙈

### CODE REVIEW

File: src/client/styles/Login.less [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        font-family: "Raleway";
+        font-weight: 900;
+        font-size: 40px;
+        margin-left: 10px;
+        color: white;
+    }
+
+    // .heading-row {
+    //     content-align: center;
+    // }
```
Comment (JZ4012) [inline src/client/styles/Login.less:133]: remove the comments

File: src/client/styles/Login.less [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        text-align: center;
+    }
+
+    .logo-image {
+        background-image: url("logo_white.png");
+        background-position: center;
+        background-size: 50px 50px;
+        width: 50px;
+        height: 50px;
+        // padding: 0px 50px;
```
Comment (JZ4012) [inline src/client/styles/Login.less:155]: same as above

File: src/client/styles/Login.less (hunk 1 above)
Comment (JZ4012) [inline src/client/styles/Login.less:133]: remove comments

File: src/client/styles/Login.less [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        // padding-right: 50px;
```
Comment (JZ4012) [inline src/client/styles/Login.less:156]: same as above

**Review by JZ4012 — CHANGES_REQUESTED** [review]

**Review by aXises — APPROVED** [review]
Nice job

**Pull request merged**